src/ontology/edits/oeo-physical.omn

```diff
@@ -7004,6 +7004,10 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/810"
     
 Class: OEO_00020003
 
+    SubClassOf: 
```

Comment:
Please make all the changes to this class in oeo-shared only.

Reply:
I'm not sure how, since 'energy transformation efficiency' and 'power' don't seem to exist in oeo-shared.

Reply:
I see. According to the current file structure, we agreed on moving everything to oeo-shared in such cases, but that is a little tricky the first time. We can do that together next week, or you can ask @fabianneuhaus tomorrow.  @nelekoehler and @h-spinde should also get to know the process.